### ISSUE TITLE
Survey Flow 

### ISSUE THREAD

**Issue by zzmarkzz321** [findthecluster/survey-app#9]
tldr; piece together everything and set up the flows for the survey.

The main page that will contain the survey

the survey page has 3 states:

1. Initial survey questions. This should either be "Feeling good!" or "Not so well" we will prompt them "How are you feeling today?"
2. Follow up questions page after selecting a choice from above. If the user selects feeling good, we collect additional meta data info (listed below) or if they listed not doing well, then we ask for symptoms.
3. If a user has selected they are not doing well, the 3 state will ask for quantitative info about their symptoms. If they are doing well, skip to state 4
4. state 4 will ask for meta data info such as sex, age, location, etc.
